silverback/_cli.py

```diff
@@ -102,6 +115,23 @@ def run(cli_ctx, account, runner_class, recorder_class, max_exceptions, path):
                 option_name="network", message="Network choice cannot support running app"
             )
 
+    if not (bots_folder := Path.cwd() / "bots").exists():
```

Comment:
This is kind of missing my suggestion, here is the order:
1. `bot:bot` (this is what `path=None` uses)
2. `bots.{path}:bot` (this is what is tried if `path` is not a valid import string)
3. `{path}` (the last override, which will just treat it as an import string e.g. `path.to.module:bot_var`)

(3) may need to be tried first, and if it fails (and `path` does not have a `.` or `:` in it), fall back to trying it as (2)